libraries/botbuilder-ai-orchestrator/tests/orchestratorAdaptiveRecognizer.test.js

```diff
@@ -10,8 +10,55 @@ const { TurnContext, MessageFactory } = require('botbuilder-core');
 const { BotFrameworkAdapter } = require('../../botbuilder/lib');
 const { StringExpression, BoolExpression, NumberExpression } = require('adaptive-expressions');
 const { NumberEntityRecognizer } = require('botbuilder-dialogs-adaptive');
+const sinon = require('sinon');
 
 describe('OrchestratorAdpativeRecognizer tests', function() {
+    it('Expect initialize is called when orchestrator obj is null', (done) => {
+        //if labelresolver is null, initialize() must be called
+        let result = [
+            {
+                score : 0.9,
+                label : {
+                    name : "mockLabel"
+                }
+            }
+        ];
+        let mockResolver = new MockResolver(result);
+        let testPaths = "test";
+
+        let rec = new OrchestratorAdaptiveRecognizer(testPaths, testPaths, mockResolver);
+        OrchestratorAdaptiveRecognizer.orchestrator = null;
+        rec.Initialize = sinon.fake();
+
+        let {dc, activity} = createTestDcAndActivity("hello");
+        rec.recognize(dc, activity)
```

Comment:
Tests are a lot cleaner with `async` syntax.

```typescript
it('Expect initialize is called when orchestrator obj is null', async () => {
        //if labelresolver is null, initialize() must be called
        const result = [
            {
                score : 0.9,
                label : {
                    name : "mockLabel"
                }
            }
        ];

        const mockResolver = new MockResolver(result);
        const testPaths = "test";

        const rec = new OrchestratorAdaptiveRecognizer(testPaths, testPaths, mockResolver);
        OrchestratorAdaptiveRecognizer.orchestrator = null;

        rec.Initialize = sinon.fake();

        const {dc, activity} = createTestDcAndActivity("hello");
        const res = await rec.recognize(dc, activity);
        assert(res);
       ...
});
```

Reply:
Thanks @joshgummersall, changed all usage of `recognize to `async` instead of promises.

Reply:
Awesome, thanks @joshgummersall , all tests updated to use async.